packages/libmypaint-gegl.rb

```diff
@@ -0,0 +1,23 @@
+require 'package'
+
+class Libmypaint < Package
+  description 'This is the brush library used by MyPaint. A number of other painting programs use it too.'
```

Comment:
Better description?  `Libmypaint is MyPaint’s brushstroke rendering code, in a form that can be used by other programs.`

Reply:
Good idea, I just used what I found on the GitHub.